lib/core/src/Cardano/Wallet.hs

```diff
@@ -1591,6 +1592,7 @@ signTx ctx wid pwd (UnsignedTx inpsNE outsNE) = db & \DBLayer{..} -> do
     nl = ctx ^. networkLayer @t
     inps = NE.toList inpsNE
     outs = NE.toList outsNE
+    md = Nothing -- no metadata
```

Comment:
Shouldn't this be better left as an argument as well ?

Reply:
Probably - fixed.